examples/api/lib/material/search_anchor/search_anchor.3.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for [SearchAnchor] that shows how to fetch the suggestions
+/// from a remote API.
+const Duration fakeAPIDuration = Duration(seconds: 1);
```

Comment:
There should be an empty line between lines 8 and 9 here.

Reply:
Done.